### ISSUE TITLE
[Feature request] Integrate with Test Adapter

### ISSUE THREAD

**Issue by nicolasnoble** [bazelbuild/vscode-bazel#186]
There is a [Test Explorer UI extension](https://marketplace.visualstudio.com/items?itemName=hbenl.vscode-test-explorer) available, which itself provides a [Test Adapter API to discover and run tests](https://github.com/hbenl/vscode-example-test-adapter). An example of such test adapter is the [Catch2 and GoogleTest adapter](https://marketplace.visualstudio.com/items?itemName=matepek.vscode-catch2-test-adapter).

It would be very neat if the vscode-bazel extension was integrating with either Test Explorer directly, or with the GoogleTest adapter, in order to provide the list of tests, and the ability to spawn a [Visual Studio Code Debugger instance](https://code.visualstudio.com/docs/cpp/cpp-debug) through the Test Explorer UI.

**Comment by haifeng-jin** [comment]
Any updates to this issue? It would be really useful for Keras contributors when they use CodeSpaces. Now they have to manually type in the paths to the tests.

**Comment by bhack** [comment]
@haifeng-jin I agree with you but it seems that more in general the main issue is that no resources are allocated on this extension:

https://github.com/bazelbuild/vscode-bazel/issues/241#issuecomment-938632324
